packages/cli/src/WebhookHelpers.ts

```diff
@@ -506,15 +506,15 @@ export async function executeWebhook(
 			responsePromise = await createDeferredPromise<IN8nHttpFullResponse>();
 			responsePromise
 				.promise()
-				.then((response: IN8nHttpFullResponse) => {
+				.then(async (response: IN8nHttpFullResponse) => {
 					if (didSendResponse) {
 						return;
 					}
 
 					const binaryData = (response.body as IDataObject)?.binaryData as IBinaryData;
 					if (binaryData?.id) {
 						res.header(response.headers);
-						const stream = Container.get(BinaryDataService).getAsStream(binaryData.id);
+						const stream = await Container.get(BinaryDataService).getAsStream(binaryData.id);
 						void pipeline(stream, res).then(() =>
 							responseCallback(null, { noWebhookResponse: true }),
 						);
```

Comment:
since the outer function is `async` now, can you please also update this to 

```suggestion
						await pipeline(stream, res);
						responseCallback(null, { noWebhookResponse: true });
```